### ISSUE TITLE
[Accessibility]: The tabbable element does not have a valid widget role

### ISSUE THREAD

**Issue by srikant-ch5** [carbon-design-system/carbon-charts#2013]
### Application/Team

SPSS 

### Environment

Browser

### What happened? What did you expect to happen instead?

Using Heatmap and few other charts we are able to see `The tabbable element does not have a valid widget role` violation.

Navigate to https://charts.carbondesignsystem.com/heatmap and scan the page using IBM Equal Access Accessibility Checker. Doing that below violations will appear.

<img width="1480" height="952" alt="Image" src="https://github.com/user-attachments/assets/6e9050dd-494f-4424-8bb6-b29c41a4f14e" />

### What WCAG 2.1 checkpoint does the issue violate?

4.1.2 Name, Role, Value

### Version

1.23.5

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P2

**Comment by srikant-ch5** [comment]
@theiliad Could someone please look into this issue ?

**Comment by theiliad** [comment]
> > Hi, what version of the library/website are you testing this on?
> > <img alt="Image" width="399" height="159" src="https://private-user-images.githubusercontent.com/14989804/469335026-02378504-8b54-4895-b3eb-246cd2126cbc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMzNzUxMTEsIm5iZiI6MTc1MzM3NDgxMSwicGF0aCI6Ii8xNDk4OTgwNC80NjkzMzUwMjYtMDIzNzg1MDQtOGI1NC00ODk1LWIzZWItMjQ2Y2QyMTI2Y2JjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzI0VDE2MzMzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjMjY5NDdiNDdmNWVkN2NiZWM3OGI3Yjc0NDgwYWZlZDU2MTZkNWM5NjBiYmVkNTMxZDUyNjg0MWE5ZWVmNzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5eMjvr5Xn5jvMDBPWtyaao_aS2tssUl7qrDYrj_bg4U">
> > those tick groups should not be tabbable as they have `tabindex="-1"`
> > pls reply with more details, happy to reopen
> 
> Hi [@theiliad](https://github.com/theiliad) I am testing this using ` IBM Equal Access Accessibility Checker.` version 4.0.6

I'm asking about the version of Carbon Charts. those tick groups that you included in your screenshot should not be tabbable as they have `tabindex="-1"`

but in your screenshot, they have `tabindex="0"`

**Comment by srikant-ch5** [comment]
Hi @theiliad any update on this ?

**Comment by srikant-ch5** [comment]
Hi @theiliad any update on this issue ? Any ETA ?

**Comment by caritaou** [comment]
This issue exists for the first element in the y/x axis, only the first item has a `tabindex=0`

<img width="1715" height="948" alt="Image" src="https://github.com/user-attachments/assets/88f25f95-fd92-4553-aa5f-3ab1f417d5d6" />

<img width="1684" height="561" alt="Image" src="https://github.com/user-attachments/assets/9fd32001-3dd7-45b3-a797-c57377158984" />

**Comment by heloiselui** [comment]
Looks like this issue still persists, so I’m going to reopen it.

**Comment by caritaou** [comment]
The Cloud Pak for Data team needs this fix in order to be fully compliant for accessibility.  Please provide the status on who is assigned to this issue and an ETA for the fix.  Any questions, you can contact Debby Nakamura, Release Manager for Cloud Pak for Data.

**Comment by amal-k-joy** [comment]
[PR](https://github.com/carbon-design-system/carbon-charts/pull/2089)

**Comment by caritaou** [comment]
Thank you @amal-k-joy for the fix, can you post what version this will be in?
